MAESTRO/Snakemake/scATAC/Snakefile

```diff
@@ -200,10 +154,21 @@ if config["platform"] == "microfluidic":
             "Result/Benchmark/%s_QCMerge.benchmark" %(config["outprefix"])
         shell:
             "python " + SCRIPT_PATH + "/scATAC_microfluidic_QC.py --log-dir {params.log} --directory {params.outdir};"
+    
+    rule scatac_celldedup:
```

Comment:
Allen in the lab is testing whether we should dedup at cell level or bulk level, and it seems dedup at bulk level is better to reduce the cutting bias. We are not sure at this point yet. How about make it as an option in the config.ymal so the users can choose to dedup at either cell level or bulk level? if the user sets dedup at cell level, it will be a symoblic link https://github.com/crazyhottommy/pyflow-single-cell/blob/feat/dedup/scATAC/rules/sc_atac_dedup.smk#L27

if user sets dedup at bulk level, we can just select one fragment from the cells with that duplicates https://github.com/crazyhottommy/pyflow-single-cell/blob/feat/dedup/scATAC/rules/sc_atac_dedup.smk#L37

Let's not `cut -f 1,2,3 {input.frag_count}` to get the bed file, we want to retain all the columns so the users can use those fragment file for other purposes.